chia/plotting/manager.py

```diff
@@ -268,6 +269,9 @@ def process_file(file_path: Path) -> Dict:
                         result.loaded_plots += 1
                         result.loaded_size += stat_info.st_size
 
+                    if file_path in self.failed_to_open_filenames:
+                        del self.failed_to_open_filenames[file_path]
```

Comment:
this is to remove files that eventually *succeed* from the failed-list, right?

Reply:
Also correct.